src/Controller/ProjectController.php

```diff
@@ -19,4 +23,27 @@ public function index(ProjectRepository $projectRepository): Response
             'projects' => $projects,
         ]);
     }
+
+    #[Route('/addProject', name: 'add')]
+    public function addProject(EntityManagerInterface $entityManager, Request $request): Response
```

Comment:
No you shouldn't use the EntityManager here, but the Repository to save your object

Reply:
change by repository and tested